packages/test-interface-criteria/src/criteria/criteriaUtils.js

```diff
@@ -287,6 +287,21 @@ function getDateAfterWhichBookingsCanBeMade(offer, opportunity) {
   return dateMinusDuration(opportunity.startDate, offer.validFromBeforeStartDate);
 }
 
+/**
+ * Get the date that the startDate - validThroughBeforeStartDate window starts
+ *
+ * @param {Offer} offer
+ * @param {Opportunity} opportunity
+ * @returns {DateTime | null} null if there is no booking window defined.
```

Comment:
```suggestion
 * @returns {DateTime | null} null if there is no booking window upper limit defined.
```

Reply:
and then likewise for the above function

Reply:
Great point, have done so